mne_bids/tests/test_write.py

```diff
@@ -34,6 +34,10 @@
                             category=ImportWarning)
     import mne
 
+try:
+    from mne.io._digitization import _get_fid_coords
+except ImportError:
+    from mne._digitization._utils import _get_fid_coords
```

Comment:
I don't like we import mne private functions outside of MNE. I would copy and adapt this function for our specific use case.